### ISSUE TITLE
Add an option to hide the "jump to top" floating button.

### ISSUE THREAD

**Issue by Tsalnor** [bluesky-social/social-app#5210]
**Is your feature request related to a problem? Please describe.**

The "jump to top" floating button is very easy to press accidentally while scrolling. If you press this, you will lose your place in the feed and all of the posts you were looking at are lost to the void, never to be found again. I have done this multiple times in the short time I have used this app, and it is starting to get frustrating.

**Describe the solution you'd like**

There should be an option to hide this floating button. Jumping to the top can still be easily done by pressing the name of the feed or pressing the home button, both of which are significantly harder to do accidentally.

**Describe alternatives you've considered**

This floating button could be removed entirely. I wouldn't miss it, though I'm sure there's someone out there who would complain if it were removed.

**Additional context**

I am on Android, but I imagine iOS has the same issue. This is not an issue for Web, but it wouldn't hurt to provide the option anyways. The compose floating button also has a similar issue, but it's not as big of a deal to accidentally press and there would need to be an alternative way to compose posts if that button were hidden.

**Comment by tlaverdure** [comment]
I agree with this issue. The floating action button on the bottom left of the screen on the feed is distracting. It covers up the faces of users and is not helpful. To scroll to the top I can tap the app header. I would rather have a different type of UI that shows up only when there is new content at the top of the feed.

**Comment by geordied** [comment]
I agree with this - if I’m scrolling with my left hand, my thumb hits it way too many times and I lose my place in the feed. Just want to toggle it off, please!

**Comment by qlik-mni** [comment]
I really agree with this one, I scroll with my left thumb and I hit this button over and over again by mistake... it's hugely annoying.

**Comment by msandin** [comment]
I really agree with this, I scroll with my left thumb and I keep hitting this button over and over again by mistake. It's really annoying, by some mile my largest gripe with the app.

**Comment by piscopo** [comment]
I would also like to agree with this issue. I just tried to start using the app again, and being someone who mostly scrolls one-handed with their left thumb, this is by far the most aggravating thing. It makes me not want to even use it. Please add an option to hide this. I can just hit the Home button if I want to jump to the top.

**Comment by ChrisRomp** [comment]
A constant problem for me on the iOS app also.
